test/relations.test.js

```diff
@@ -709,40 +710,142 @@ describe('relations', function() {
       }
     });
 
-    it('should fetch scoped instances with paging filters', function(done) {
-      Physician.create(function(err, physician) {
-        physician.patients.create({name: 'a'}, function() {
-          physician.patients.create({name: 'z'}, function() {
-            physician.patients.create({name: 'c'}, function() {
-              verify(physician);
+    describe('fetch scoped instances with paging filters', function() {
+      var samplePatientId;
+      var physician;
+
+      beforeEach(createSampleData);
+
+      context('with filter skip', function() {
+        it('skips the first patient', function(done) {
+          physician.patients({skip: 1}, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(2);
+            ch[0].name.should.eql('z');
+            ch[1].name.should.eql('c');
+            done();
+          });
+        });
+      });
+      context('with filter order', function() {
+        it('orders the result by patient name', function(done) {
+          physician.patients({order: 'name DESC'}, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(3);
+            ch[0].name.should.eql('z');
+            ch[2].name.should.eql('a');
```

Comment:
I think if checks for length, `z` and `a` pass, there is no possibility that the one in middle is not `c` :p but I can add it anyway